src/test/scala/akka/persistence/jdbc/journal/JdbcJournalSpec.scala

```diff
@@ -19,11 +19,11 @@ package akka.persistence.jdbc.journal
 import akka.persistence.CapabilityFlag
 import akka.persistence.jdbc.config._
 import akka.persistence.jdbc.util.Schema._
-import akka.persistence.jdbc.util.{ClasspathResources, DropCreate, SlickDatabase, SlickExtension}
+import akka.persistence.jdbc.util.{ ClasspathResources, DropCreate, SlickDatabase, SlickExtension }
 import akka.persistence.journal.JournalSpec
-import com.typesafe.config.{Config, ConfigFactory, ConfigValueFactory}
+import com.typesafe.config.{ Config, ConfigFactory, ConfigValueFactory }
 import org.scalatest.concurrent.ScalaFutures
-import org.scalatest.{BeforeAndAfterAll, BeforeAndAfterEach}
+import org.scalatest.{ BeforeAndAfterAll, BeforeAndAfterEach }
```

Comment:
does scalariform not run on compile? 
Or I guess after rebasing the settings changed?

Reply:
have you run `test:compile`? Most of the files that were reformatted were test files. Except one that was already in master and got merged here when I resolved the conflict